### ISSUE TITLE
Validate installed node version (and npm/yarn version?) against engine definition

### ISSUE THREAD

**Issue by voxpelli** [voxpelli/node-installed-check#2]
As stated in title – currently only installed modules are checked against the required version ranges, engines are not at all verified against any installed versions.

**Comment by flipxfx** [comment]
+1 I installed this package thinking it would check my installed node/npm version. I think the current checks are useful, but adding that last check against the current installed node/npm against engines would be awesome.

**Comment by flippidippi** [comment]
Currently we are using this package along with https://github.com/kruppel/check-engines.
I think adding that logic into node-installed-check would make sense and cover all the checks.

**Comment by voxpelli** [comment]
I haven't forgotten about this, would like to get it in soon 🙈

**Comment by flippidippi** [comment]
Any chance this is still looking to be added?

**Comment by bcomnes** [comment]
Maybe an easy way to do this would be to references the version npm that shipped with the lowest version of node, if that data is roughly available.

**Comment by flippidippi** [comment]
I ended up making a new package that did everything I needed which checks installed engines and server of node modules installed.

https://www.npmjs.com/package/check-installed

**Comment by voxpelli** [comment]
Right, but isn’t this more or less built into package managers nowadays?
